src/containers/BookForm.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: null,
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit = event => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
- [ ] We can remove this Eslint disable line and add a type checking to the component:
```JS
BookForm.propTypes = {
  createBook: PropTypes.func.isRequired,
};
```